lesson05/assignment/css/_layout.partial.scss

```diff
@@ -0,0 +1,113 @@
+/*
+ * This defines the grid that's used to display that parts of the webpage.
+ * It also defines some information about the font that's constant for all
+ * parts of the site.
+ */
+
+body {
+    color: $white;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 32px;
```

Comment:
In responsive design, we use relative sizing units (rem, %) rather than static unit px, with very few exceptions. In font sizing, it's especially important, because setting a static size can have a real impact on low vision users. This article gives a good overview: https://css-tricks.com/accessible-font-sizing-explained/#think-about-proportions-not-size